tests/src/Functional/CorporateFooterRenderTest.php

```diff
@@ -802,11 +862,14 @@ protected function assertFooterPresence(string $branding, int $expected_section_
    *   Ecl branding, core/standardised.
    * @param array $expected
    *   The expected data.
+   * @param bool $icon
+   *   If the link should have an icon, defaults to FALSE.
    */
-  protected function assertListLink(NodeElement $actual, string $branding, array $expected): void {
+  protected function assertListLink(NodeElement $actual, string $branding, array $expected, bool $icon = FALSE): void {
```

Comment:
This method shouldn't be changed because the fact that there is an icon doesn't ensure that is the external one and the core branding doesn't support site specific footer links anyway.

We can revert the changes for this method and simply assert the external icon is present or not in the test method.